design/authenticators-status/authn-status-new-implem.md

```diff
@@ -1,65 +1,84 @@
 
-_For a complete status check flow [diagram](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn-status-flow.jpeg)_
+If the configuration was successful, the response received will reflect this. If not, an error will be provided in the response, identifying the step of failure.
 
-This doc describes how to expand the status check functionality to other authenticators.
+The first authenticator to utilize this feature xis Authn OpenID Connect. For more information about the design behind the Status API and implementation details see the [Status epic](https://github.com/cyberark/conjur/issues/1062). 
+It is advised that you read the epic before expanding status to other authenticators.
+
+_For a complete status flow [diagram](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn-status-flow.jpeg)_
+
+This doc describes how to expand the status functionality to other authenticators.
 For each authenticator, a suite of [general](https://github.com/cyberark/conjur/blob/master/design/authenticators-status/authn_status_general.md) 
 checks are run followed by those specific to the authenticator.  
 
 General checks include:
-1. The requesting user has access to the authenticator status route webservice
-1. _The authenticator implements the status check_
+
+**NOTE:** Steps that require developer intervention to implement and enable status for authenticators are italicized below
+1. The requesting user has access to the authenticator status webservice
+    1. See the security section [here](https://github.com/cyberark/conjur/issues/1062) for more info about this webservice 
+1. _The authenticator implements the status method_
 1. The account exists
 1. The webservice exists
 1. The authenticator is enabled in the ENV
-1. _Checks specific for the new authenticator_
+1. _Checks specific requirements for the new authenticator_
 
-To properly implement the status check functionality for the new authenticator, you will need to expand on italicized checks 2 and 6. 
-As described in step 2, the status of an authenticator method is checked for implementation. Therefore, you will need to 
-create a new class in the structure of Authentication::<New_Authenticator>::Authenticator and add a status method like so:
+To properly implement the status functionality in the new authenticator, you will need to expand on italicized checks 2 and 6. 
+As described in step 2, the status of an authenticator method is checked for implementation. Therefore, you will need to add a status method to the Authenticator class (`Authentication::Authn<Type>::Authenticator`) like so:
 
 ```ruby
-def status(account:, authenticator_name:, webservice:)
-  Authentication::<New_Authenticator>::ValidateStatus.new.(
-    account: account,
-      service_id: webservice.service_id
+def status(authenticator_status_input:)
+  Authentication::Authn<Type>::ValidateStatus.new.(
+    account: authenticator_status_input.account,
```

Comment:
i wouldn't explicitly write "account" & "service_id" as `authenticator_status_input` includes other parameters too. maybe we can write them down and say that the developer can use any of them but doesn't have to use all of them in the `ValidateStatus` command class. the parameters are:
account, service_id, authenticator_name, username, webservice, status_webservice.

I think it's best to add descriptions on each one, i'll let you explain it on your own :)